content/en/docs/rules/custom-ruleset.md

```diff
@@ -0,0 +1,383 @@
+
+- The `desc` key provides a detailed description of the rule's purpose, behavior, or the events it aims to detect. It helps with understanding the rule's intent and assists in documentation. Missing this key in the rule will make Falco show the error message:
+
+  ```
+  LOAD_ERR_YAML_VALIDATE (Error validating internal structure of YAML file): Item has no mapping for key 'desc'
+  ```
+- The `priority` key represents the severity of the alert triggered by Falco and corresponds with the well-known Syslog severities: emergency, alert, critical, error, warning, notice, informational, debug. Missing this key in the rule will make Falco show the error message:
+
+  ```
+  LOAD_ERR_YAML_VALIDATE (Error validating internal structure of YAML file): Item has no mapping for key 'priority'
+  ```
+- The `condition` key defines the conditions that need to be satisfied for the rule to trigger an alert. It consists of one or more expressions or statements that evaluate to true when the desired event occurs. Missing this key in the rule will make Falco show the error message:
+
+  ```
+  LOAD_ERR_YAML_VALIDATE (Error validating internal structure of YAML file): Item has no mapping for key 'condition'
+  ```
+- The `output` key determines the output format of the alert generated by the rule. It specifies how the alert should be formatted and what information should be included,. Missing this key in the rule will make Falco show the error message:
+
+  ```
+  LOAD_ERR_YAML_VALIDATE (Error validating internal structure of YAML file): Item has no mapping for key 'output'
+  ```
+
+- The `tags` key is used to categorize the ruleset into groups of related rules. Although not mandatory when starting to write a rule, its use is highly recommended at a later stage for management purposes. For further information refer to the [Tags section of the Style Guide of Falco Rules][/docs/rules/style-guide/#tags)
+
+To enhance your Falco rules further, refer to the [Advanced Rule Syntax](/docs/rules/basic-elements/#advanced-rule-syntax) documentation. The [Style Guide of Falco Rules](/docs/rules/style-guide) is also a highly recommended document to ensure your rules are easier to maintain and share with the community. These resources will provide you with valuable information about additional keys that can be used to augment and customize your Falco rules. Exploring these advanced options will allow you to expand the capabilities and effectiveness of your rules.
+
+## Building up the Condition
+
+A condition in Falco acts as a checklist of requirements that an event must meet in order for Falco to evaluate the condition as true. To comprehend this evaluation process, it is essential to have a grasp of [Boolean algebra](https://en.wikipedia.org/wiki/Boolean_algebra). Ultimately, the condition will either evaluate to true, triggering the associated alert and bypassing the remaining rules, or evaluate to false, causing the next rule to be evaluated.
```

Comment:
> A condition in Falco acts as a checklist of requirements that an event must meet in order for Falco to evaluate the condition as true.

This reads a bit like a circular definition -- using condition to define a condition.